Wrappers/Python/cil/optimisation/functions/TotalVariation.py

```diff
@@ -91,6 +92,11 @@ class TotalVariation(Function):
 
         .. math:: \underset{u}{\mathrm{argmin}} \frac{1}{2\frac{\tau}{1+\gamma\tau}}\|u - \frac{b}{1+\gamma\tau}\|^{2} + \mathrm{TV}(u) 
 
+    warmstart : :obj`boolean`, default = False
+        If set to true, the FGP aglorithm to calculate the TV proximal is initiated by the final value from the previous iteration and not at zero. 
+        This allows the max_iteration value to be reduced to 5-10 iterations. 
+
```

Comment:
```suggestion
With warmstart this function will keep in memory the range of the gradient of the image to be denoised, i.e. N times the dimensionality of the image. However, during the evaluation of `proximal` the memory requirements will be unchanged as the same amount of memory will need to be allocated and deallocated. 
```

Reply:
Thanks - will change this!

Reply:
The peak memory is not going to increase with this, it is the idle memory that's going to be affected.